test/dish_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/dish'
+require './lib/potluck'
+require 'pry'
+
+class DishTest < Minitest::Test
+
+  # def test_it_exists
+  #   dish = Dish.new
+  #   assert_instance_of Dish, dish
+  # end
```

Comment:
Interesting! I would recommend adjusting this test to pass an argument rather than commenting it out.